### PR TITLE
Significantly improved the introduction

### PR DESCRIPTION
**Pull request by Gitoffthelawn**

### CODE REVIEW

**Comment by duythien** [comment]
@Gitoffthelawn Thanks

**Comment by Gitoffthelawn** [comment]
@duythien You're welcome! :-)

File: introduction.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
 ## Welcome!
 
-Welcome to Phanbook. Our missing is to give you and advanced tool for developer created fast website Questions and Answers site also create blog easy with PHP.
+Welcome to Phanbook. Our mission is to give you an advanced tool for creatinh fast Questions and Answers websites and also creating blogs easily with PHP.
```
Comment (duythien) [inline introduction.md]: creatinh should be created, is right?

Comment (Gitoffthelawn) [inline introduction.md]: Yes.  Good catch!  I have made the correction.

**Pull request merged**